src/providers/MeetingProvider/docs/MeetingManager.stories.mdx

```diff
@@ -118,6 +118,14 @@ const MyApp = () => {
 
 2. Share the same `MeetingManager` instance with multiple different `MeetingProvider`s.
 
+**IMPORTANT NOTE**
+This approach has limitation though. This should be used only in very specific cases where you want to share the meeting manager instance
+variable values and you will not use video, roster and audio at both places. Still, the hooks and providers may not work as expected as `MeetingProvider` wraps all the other providers.
+The video will not work at both places due to limitations of JS SDK to bind a video stream to a single video tile only. Please check [this](https://github.com/aws/amazon-chime-sdk-component-library-react/issues/492#issuecomment-846317339) issue for more details.
+
+Also, you have to use different `attendeeInfo` when joining the meeting, otherwise, the first attendee who joined the meeting will leave the meeting with [AudioJoinedFromAnotherDevice](https://aws.github.io/amazon-chime-sdk-js/enums/meetingsessionstatuscode.html#audiojoinedfromanotherdevice)
```

Comment:
You do not have to join again right? Could we clarify that they do not need to call join again? And if they do make sure to use a different `attendeeInfo`

Reply:
Yup right, thanks for the catch. Have updated it to:
If you are using the same `attendeeInfo`, then please make sure your app calls `meetingManager.join()` only once for that `attendeeInfo`. You can join a meeting only once with the same `attendeeInfo`. If you want to join the meeting again, you have to use different `attendeeInfo`, otherwise, the first attendee who joined the meeting will leave the meeting with [AudioJoinedFromAnotherDevice](https://aws.github.io/amazon-chime-sdk-js/enums/meetingsessionstatuscode.html#audiojoinedfromanotherdevice) meeting session status code.